.github/actions/run-tests/action.yaml

```diff
@@ -17,6 +17,13 @@ runs:
       with:
         version: 8.9.0
 
+    # Set up the necessary env vars for the test suite.
+    - run: echo "ANS_TEST_ACCOUNT_PRIVATE_KEY=0x37368b46ce665362562c6d1d4ec01a08c8644c488690df5a17e13ba163e20221" >> .env
+      shell: bash
+    - run: echo "ANS_TEST_ACCOUNT_ADDRESS=0x585fc9f0f0c54183b039ffc770ca282ebd87307916c215a3e692f2f8e4305e82" >> .env
+      shell: bash
```

Comment:
@0xmaayan I am not sure if this is correct, will test in CI

Reply:
Where I moved the ANS installation to, you won't need this.